project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,12 @@
+import sbt.Setting
+import scoverage.ScoverageKeys.{coverageHighlighting, _}
+
+object CodeCoverageSettings {
+  val settings: Seq[Setting[?]] = Seq(
+    coverageExcludedFiles := "<empty>;Reverse.*;.*filters.*;.*components.*;.*repositories.*;" +
```

Comment:
you just need 

coverageExcludedFiles := "<empty>;.*components.*;.*Routes.*;",

here

Reply:
".*components.*;.*repositories.*;.*Routes.*",